example/src/domain/Full.js

```diff
@@ -128,3 +131,54 @@ function Conditional({ field, children, reverse = false }) {
   const value = useFormFieldValue(field)
   return (reverse ? !value : value) && children
 }
+
+function useSendSignalWhenIsVisited(form, f) {
+  const callbackRef = React.useRef(f)
+  React.useEffect(
+    () => {
+      const unsubscribe = snapshot.subscribeToFieldState(
```

Comment:
Couldn't you also do 
```
const unsubscribe = form.state.subscribe(() => {
  if (getIsVisited(form)) {
    callbackRef.current()
    unsubscribe()
  }
})
```

Or does this not subscribe to state changes in children?
What is the difference exactly?

Reply:
Great question. `field.state` gives you the state of the field itself. So the subscription is only to changes of that particular field.

`snapshot` subscribes to changes in the `field` state and (if applicable) changes in it's children (recursively).

Your questions lead to me believe the new name is poorly choosen. I think we should rename it to `subscribeToField`.